src/main/msp/msp.h

```diff
@@ -19,6 +19,15 @@
 
 #include "common/streambuf.h"
 
+#define MSP_V2_FRAME_ID         255
+
+typedef enum {
+    MSP_V1          = 0,
+    MSP_V2_OVER_V1  = 1,
+    MSP_V2_NATIVE   = 2,
+    MSP_VERSION_COUNT
+} mspVersion_e;
```

Comment:
`#define MSP_VERSION_MAGIC_INITIALIZER { 'M', 'M', 'X' }` for message codes here, it's much easier to keep in sync with enum values ...

Reply:
Agreed.